hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableConfig.java

```diff
@@ -81,7 +81,7 @@
 
   public static final ConfigProperty<HoodieTableVersion> HOODIE_TABLE_VERSION_PROP = ConfigProperty
       .key("hoodie.table.version")
-      .defaultValue(HoodieTableVersion.ZERO)
+      .defaultValue(HoodieTableVersion.ONE)
```

Comment:
can you ensure this is compatible on an existing table at version 0?

Reply:
yes, HoodieTableMetaClient.PropertyBuilder set table version as below.
```
tableConfig.setValue(HoodieTableConfig.HOODIE_TABLE_VERSION_PROP,
          String.valueOf(HoodieTableVersion.current().versionCode()));
```
So, this default value is actually of no use. But will see how to test this out. 
I guess, I might have to generate a table w/ 0.5.2. And then try to use this patch and see if upgrade happens. Upgrade here refers to tableversion 0 to 1 which happened w/ 0.6.0.

Reply:
I don't see the upgrade kicking in actually. So, here is what is happening. 
if hoodie.properties does not have any table version, this default value kicks in. 
So, for a table created with 052, if I use this patch to trigger an insert, I don't see an upgrade kicking in. bcoz, the hoodie.props will not have any table version and so this default gets picked up. 

Looks like, we might have to leave the default version as is.

Reply:
lets leave it as is and async this after